angular/projects/catalyst/src/lib/directives/proxies.ts

```diff
@@ -227,6 +227,27 @@ export declare interface CatDatepicker extends Components.CatDatepicker {
 }
 
 
+@ProxyCmp({
+})
+@Component({
+  selector: 'cat-datepicker-inline',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: '<ng-content></ng-content>',
+  // eslint-disable-next-line @angular-eslint/no-inputs-metadata-property
+  inputs: [],
+})
+export class CatDatepickerInline {
+  protected el: HTMLElement;
+  constructor(c: ChangeDetectorRef, r: ElementRef, protected z: NgZone) {
+    c.detach();
+    this.el = r.nativeElement;
+  }
+}
+
+
```

Comment:
I think you should remove extra spaces here

Reply:
These are auto generated by the stencil toolchain